site/cds_rdm/schemes.py

```diff
@@ -0,0 +1,57 @@
+import re
+
+
+def cds_reference_number():
+    """Define validator for `custom_scheme`."""
+    return {
+        "validator": lambda value: True,
+        "normalizer": lambda value: value,
+        "filter": ["cds_ref"],
+        "url_generator": None,
+    }
+
+
+
+aleph_regexp = re.compile(r"\d+CER$", flags=re.I)
+
+
+def is_aleph(val):
+    """Test if argument is a PubMed ID.
+
+    Warning: PMID are just integers, with no structure, so this function will
+    say any integer is a PubMed ID
+    """
+    return aleph_regexp.match(val)
+
+
+def aleph():
+    """Define validator for `custom_scheme`."""
+    return {
+        "validator": is_aleph,
+        "normalizer": lambda value: value,
```

Comment:
you don't have to pass this if it is the same as the default [here](https://github.com/inveniosoftware/idutils/blob/master/idutils/ext.py#L58)